db/seeds.rb

```diff
@@ -1,7 +1,19 @@
 require 'faker'
 
+# 50.times do
```

Comment:
Please delete all the commented-out code in this file

Reply:
We need it there because the various machines in the group are always consistently seeded.